src/components/transactions/transactions.js

```diff
@@ -5,12 +5,45 @@ import { BarChartComponent } from "../bar-chart/bar-chart";
 import { TableTransactionsComponent } from "../table-transactions/table-transactions";
 import { AddTransactionComponent } from "../add-transaction-dialog/add-transaction-dialog";
 
+import { TransactionListService } from "../../services/transaction-service";
+
 export class TransactionsComponent {
   constructor(mountPoint, props) {
     this.mountPoint = mountPoint;
     this.props = props;
   }
 
+  getStoredData() {
+    TransactionListService.get()
+      .then(list => this.showStoredTransactions(list))
+      .catch(e => console.error(`get transactions: ${e.message}`));
+  }
+
+  setStoredData(data) {
+    this.list.unshift(data);
+    TransactionListService.set(this.list)
+      .then(() => {})
+      .catch(e => console.error(`set transactions: ${e.message}`));
+  }
+
+  showStoredTransactions(list) {
```

Comment:
default function agruments
```js
showStoredTransactions(list = []) { . }
```

Reply:
It doesn't work because default is suitable only for undefined value. Here is null